java/daikon/DynComp.java

```diff
@@ -122,6 +142,38 @@ public class DynComp {
   /** Synopsis for the DynComp command line. */
   public static final String synopsis = "daikon.DynComp [options] target [target-args]";
 
+  /**
+   * Locate the classfile file that a class was loaded from. The result is a URL for the file
+   * (starting with "file:" or "jar:file:").
+   *
+   * @param c class to look up
+   * @return URL to the classfile, or null if not found
```

Comment:
If whereLoadedFrom is seen as a general utility method, then it could return null if the class could not be located do to the use of a custom classloader.  It cannot happen in our case.  Do we want to keep general code or custom tailor to our use?

Reply:
Given a custom classloader, wouldn't `getResource()` still work?